working-examples/reflow-carousel-panel-horizontal-scroll/index.html

```diff
@@ -0,0 +1,304 @@
+<!DOCTYPE html>
+<html lang="en" xml:lang="en" xmlns="http://www.w3.org/1999/xhtml">
+    <head>
+        <title>Carousel panels that scroll horizontally are designed to fit within a width of 320 CSS pixels on a vertically scrolling page</title>
+        <link rel="stylesheet" type="text/css" href="../../css/sources.css" class="remove"/>
+        <style>
+            * {
+                margin: 0;
+                padding: 0;
+                box-sizing: border-box;
+            }
+
+            body {
+                font-family: sans-serif;
+                margin: 0;
+                padding: 0;
+                color: #000;
+            }
+
+            .carousel {
+                position: relative;
+                overflow: hidden;
+            }
+
+            h2 {
+              margin: 20px;
+            }
+            .carousel .prod-list > ul {
+                display: flex;
+                gap: 20px;
+                scroll-behavior: smooth;
+                overflow-x: scroll;
+                margin: 0;
+                padding: 0 20px;
+            }
+
+            .carousel .prod-list > ul li {
+                list-style: none;
+                flex: 0 0 320px;
+                background-color: #efefef;
+                border: 1px solid #ccc;
+                border-radius: 8px;
+                display: block;
+                max-width: 320px;
+              padding:20px;
+            }
```

Comment:
Make the scrolling nicer by "snapping" the cards (part 2 of 2):
```suggestion
              padding:20px;
              scroll-snap-align: center;
            }
```